### PR TITLE
Solved issue #115 Made changes in Apply.js so that it becomes responsive

### PR DESCRIPTION
**Pull request by varun14gb**
Made changes in Apply.js so that it becomes responsive

before :
Screenshot (93)

after : 
Screenshot (92)

### CODE REVIEW

File: src/views/pages/common/Apply.js [hunk 1]
```diff
@@ -62,7 +63,6 @@ const useStyles = makeStyles(theme => ({
 
 function Apply({ className, ...rest }) {
   const classes = useStyles();
-
```
Comment (Abhishek-kumar09) [inline src/views/pages/common/Apply.js:65]: please restore it.

Comment (varun14gb) [reply src/views/pages/common/Apply.js:65]: Done

Comment (Abhishek-kumar09) [reply src/views/pages/common/Apply.js:65]: You have not restored the linebreak, please enter a new here.  Your changes are approvable.

Comment (Abhishek-kumar09) [reply src/views/pages/common/Apply.js:65]: @varun14gb  You have not restored the linebreak, please enter a new line here.  Your changes are approvable.

Comment (varun14gb) [reply src/views/pages/common/Apply.js:65]: @Abhishek-kumar09 I am sorry that first time I didn't understood what you were asking me to do.

Comment (Abhishek-kumar09) [reply src/views/pages/common/Apply.js:65]: No problem @varun14gb.

**Review by Abhishek-kumar09 — APPROVED** [review]

**Review by KeenWarrior — APPROVED** [review]

**Pull request merged**